playbooks/ansible.cfg

```diff
@@ -0,0 +1,45 @@
+[defaults]
```

Comment:
I don't think we need this file. If we run the playbooks with the `openstack-ansible` command line wrapper everything that we should need should be provided, from a library, module, role perspective.

Reply:
Was having issues running playbooks from within the rpc-upgrades repo (couldn't find inventory file) which is why I added it.  Once I added it, it seemed to link up everything correctly.

Reply:
OK